themes/theme-ios11/pages/Product/components/Media/index.jsx

```diff
@@ -1,8 +1,9 @@
 import React, { Fragment, memo } from 'react';
 import PropTypes from 'prop-types';
 import { isBeta } from '@shopgate/engage/core';
-import { MediaSlider, ProductImage } from '@shopgate/engage/product';
-import ImageSlider from './components/ImageSlider';
+import { ProductImage } from '@shopgate/engage/product';
+import ProductImageSlider from './components/ProductImageSlider';
+import ProductMediaSlider from './components/ProductMediaSlider';
 import connect from './connector';
```

Comment:
The connector is removed in `theme-gmd`. Does it need here?

Reply:
Nope. I've also removed it here.